samples/rawvm/README.md

```diff
@@ -0,0 +1,158 @@
+
+# RawVM in Istio 0.2 demo notes
+
+## MySQL Installation:
+
+### Official oracle version
+```shell
+wget https://dev.mysql.com/get/mysql-apt-config_0.8.7-1_all.deb
+sudo dpkg -i mysql-apt-config_0.8.7-1_all.deb
+# Select server 5.7 (default), tools and previews not needed/disabled
+sudo apt-get update
+sudo apt-get install mysql-server
+# Clearly this is insecure, don't do that for prod !
+sudo mysql
+   ALTER USER 'root'@'localhost' IDENTIFIED WITH mysql_native_password BY 'password';
+   # Remote 'root' can read test.*, to avoid
+   # ERROR 1130 (HY000): Host '...' is not allowed to connect to this MySQL server
+   create user 'root'@'%' IDENTIFIED WITH mysql_native_password BY 'password';
+   GRANT SELECT ON test.* TO 'root'@'%';
+# Create tables :
+mysql -u root -h 127.0.0.1 --password=password < ~/github/istio/samples/apps/bookinfo/src/mysql/mysqldb-init.sql
+# And to be able to connect remotely (only needed to test before injection)
+sudo vi /etc/mysql/mysql.conf.d/mysqld.cnf
+# comment out:
+#bind-address   = 127.0.0.1
+sudo service mysql restart
+# check it's now binding on *
+$ sudo lsof -i :3306
+COMMAND   PID  USER   FD   TYPE DEVICE SIZE/OFF NODE NAME
+mysqld  29145 mysql   31u  IPv6 168376      0t0  TCP *:mysql (LISTEN)
+# from another host, verify it works:
+ldemailly@benchmark-2:~$ mysql -u root -h instance-1 --password=password test -e "select * from ratings"
+mysql: [Warning] Using a password on the command line interface can be insecure.
++----------+--------+
+| ReviewID | Rating |
++----------+--------+
+|        1 |      5 |
+|        2 |      6 |
++----------+--------+
+# for low volume troubleshooting:
+mysql -u root -h instance-1 --password=password
+  SET global general_log_file='/tmp/mysqlquery.log';
+  SET global general_log = 1;
+# then
+tail -f /tmp/mysqlquery.log  
+```
+
+### Or
+sudo apt-get mariadb-server
```

Comment:
what is the purpose for mariadb-server?

Reply:
It's easier to install/more 'open source'